emails/tests/utils_tests.py

```diff
@@ -103,3 +106,73 @@ def test_get_domains_from_settings(self):
             'RELAY_FIREFOX_DOMAIN': 'default.com',
             'MOZMAIL_DOMAIN': 'test.com'
         }
+
+@override_settings(SITE_ORIGIN='https://test.com')
+@patch('emails.utils.GENERAL_TRACKERS', ['open.tracker.com'])
+@patch('emails.utils.STRICT_TRACKERS', ['strict.tracker.com'])
+class RemoveTrackers(TestCase):
+    def setUp(self):
+        self.expected_content = (
+            '<a href="https://test.com/faq">A link</a>\n'
+            + '<img src="https://test.com/faq">An image</img>'
+        )
+
+    def test_simple_general_tracker_replaced_with_relay_content(self):
+        content = (
+            '<a href="https://open.tracker.com/foo/bar.html">A link</a>\n'
+            + '<img src="https://open.tracker.com/foo/bar.jpg">An image</img>'
+        )
+        changed_content, general_removed, general_count, strict_count = remove_trackers(content)
+
+        assert changed_content == self.expected_content
+        assert general_removed == 2
+        assert general_count == 2
+        assert strict_count == 0
+    
+    def test_complex_general_tracker_replaced_with_relay_content(self):
+        content = (
+            '<a href="https://foo.open.tracker.com/foo/bar.html">A link</a>\n'
+            + '<img src="https://bar.open.tracker.com/foo/bar.jpg">An image</img>'
+        )
+        changed_content, general_removed, general_count, strict_count = remove_trackers(content)
+
+        # assert changed_content == self.expected_content
```

Comment:
This assertion works, can be uncommented.
```suggestion
        assert changed_content == self.expected_content
```